scene/gui/graph_edit_arranger.cpp

```diff
@@ -437,63 +435,61 @@ float GraphEditArranger::_calculate_threshold(const StringName &p_v, const Strin
 	float threshold = p_current_threshold;
 	if (p_v == p_w) {
 		int min_order = MAX_ORDER;
-		GraphEdit::Connection incoming;
-		List<GraphEdit::Connection> connection_list;
-		graph_edit->get_connection_list(&connection_list);
-		for (List<GraphEdit::Connection>::Element *E = connection_list.front(); E; E = E->next()) {
-			if (E->get().to_node == p_w) {
-				ORDER(E->get().from_node, r_layers);
+		Ref<GraphEdit::Connection> incoming;
+		List<Ref<GraphEdit::Connection>> connection_list = graph_edit->get_connection_list();
+		for (List<Ref<GraphEdit::Connection>>::Element *E = connection_list.front(); E; E = E->next()) {
```

Comment:
We can iterate lists with something like

```suggestion
		for (cosnt Ref<GraphEdit::Connection> &connection : connection_list) {
```